### PR TITLE
Add option values plugin type. Fixes issue 77

### PR DESCRIPTION
**Pull request by sjrd218**
Adding plugin to provide option list values

### CODE REVIEW

**Comment by sjrd218** [comment]
https://github.com/rundeck/docs/pull/201 documentation

File: rundeckapp/grails-app/assets/javascripts/menu/joboptions.js [hunk 1]
```diff
@@ -331,14 +337,23 @@ function Option(data) {
...
             ko.utils.arrayForEach(remotevalues, function (val) {
                 arr.push(val);
             });
-        } else if (localvalues != null) {
+        } else if (self.hasValues()) {
             ko.utils.arrayForEach(localvalues, function (val) {
                 arr.push(new OptionVal({label: val, value: val}));
             });
+        } else if (self.hasPluginValues()) {
+            console.log(pluginvalues)
```
Comment (gschueler) [inline rundeckapp/grails-app/assets/javascripts/menu/joboptions.js]:
```suggestion
```

File: rundeckapp/grails-app/assets/javascripts/menu/joboptions.js [hunk 2]
```diff
@@ -331,14 +337,23 @@ function Option(data) {
...
+            ko.utils.arrayForEach(pluginvalues, function (val) {
+                console.log(val.name)
```
Comment (gschueler) [inline rundeckapp/grails-app/assets/javascripts/menu/joboptions.js]:
```suggestion
```

File: rundeckapp/grails-app/assets/javascripts/menu/joboptions.js [hunk 3]
```diff
@@ -331,14 +337,23 @@ function Option(data) {
...
+                console.log(val.value)
```
Comment (gschueler) [inline rundeckapp/grails-app/assets/javascripts/menu/joboptions.js]:
```suggestion
```

File: rundeckapp/grails-app/domain/rundeck/Option.groovy [hunk 4]
```diff
@@ -251,6 +258,10 @@ public class Option implements Comparable{
         }else{
             opt.secureExposed=false
         }
+        println "optionValuesPluginType: ${data.optionValuesPluginType}"
```
Comment (gschueler) [inline rundeckapp/grails-app/domain/rundeck/Option.groovy]:
```suggestion
```

**Pull request merged**